cg/models/pipeline_qc_metrics/pipeline_qc_metrics.py

```diff
@@ -0,0 +1,113 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class LaneMetric(BaseModel):
+    """Model for lane specific sequencing metrics."""
+
+    lane: int
+    number_of_reads: int
+    _yield: int | None = None
+
```

Comment:
I don't know exactly how deep we intend to go with this trending, but would it at all be relevant to build a database on "overrepresented_sequences" from for instance fastp:
```
          "overrepresented_sequences": {
                "AATGGAATGGAATGGAATGGAATGGAATGGAATGGAATGG": 573,
                "ACACACACACACACACACACACACACACACACACACACAC": 892,
                "ATGGAATGGAATGGAATGGAATGGAATGGAATGGAATGGA": 124,
                "ATTCCATTCCATTCCATTCC": 6876,
                "CATTCCATTCCATTCCATTC": 4596,
                "CCATTCCATTCCATTCCATT": 1445,
                "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC": 24,
                "GAATGGAATGGAATGGAATGGAATGGAATGGAATGGAATG": 279,
                "GGAATGGAATGGAATGGAATGGAATGGAATGGAATGGAAT": 900,
                "TCCATTCCATTCCATTCCAT": 2243,
                "TGGAATGGAATGGAATGGAATGGAATGGAATGGAATGGAA": 437,
                "TTCCATTCCATTCCATTCCA": 4918,
                "TTTTTTTTTTTTTTTTTTTT": 9313
            }
```
Just in case someone wants to dig into it at some point? Though I don't have a clear use case scenario in mind at the moment.